202311 lms-refactor (#844)
utelecon/utelecon.github.io: src/pages/en/itc_lms/index.md

---
title: ITC-LMS
breadcrumb:
   title: ITC-LMS
redirect_from:
   - /en/lms_lecturers/login
   - /en/lms_students
---

## What is ITC-LMS

[ITC-LMS](https://itc-lms.ecc.u-tokyo.ac.jp/) is a learning management system of the University of Tokyo. Features of ITC-LMS include distribution of lecture materials, submission of assignments, online examinations, and notification from faculty members to students.

> For using ITC-LMS, [UTokyo Account](/utokyo_account/) is required. If you have not yet started using UTokyo Account, please read "[Getting Ready for ICT Systems at UTokyo](/en/oc/) (for New Students)" or "[Getting Ready for ICT Systems at the UTokyo](/en/faculty_members/) (for Faculty Members)" at first.

Reviewer: ```suggestion
For using ITC-LMS, [UTokyo Account](/utokyo_account/) is required. If you have not yet started using UTokyo Account, please read "[Getting Ready for ICT Systems at UTokyo](/en/oc/) (for New Students)" or "[Getting Ready for ICT Systems at the UTokyo](/en/faculty_members/) (for Faculty Members)" first.
```

at first は「初めのうちは」「最初は」という意味です

[ITC-LMS](https://itc-lms.ecc.u-tokyo.ac.jp/) is a learning management system of the University of Tokyo. Features of ITC-LMS include distribution of lecture materials, submission of assignments, online examinations, and notification from faculty members to students.

> For using ITC-LMS, [UTokyo Account](/utokyo_account/) is required. If you have not used UTokyo Account yet, please read "[Getting Ready for ICT Systems at UTokyo](/en/oc/) (for New Students)" or "[Getting Ready for ICT Systems at the UTokyo](/en/faculty_members/) (for Faculty Members)" at first.

Reviewer 2: ```suggestion
For using ITC-LMS, [UTokyo Account](/utokyo_account/) is required. If you have not yet started using UTokyo Account, please read "[Getting Ready for ICT Systems at UTokyo](/en/oc/) (for New Students)" or "[Getting Ready for ICT Systems at the UTokyo](/en/faculty_members/) (for Faculty Members)" at first.
```

have usedに違和感があったので，こうしてみました．

### Setting notification

You can accept the notification from ITC-LMS by email or LINE application.

> To set up the notification, please read "[Set the notifications](notification/)".

Reviewer 2: ```suggestion
To set up the notification, follow the setup procedures in the "[Set the notifications](notification/)" page.
```
https://utelecon.adm.u-tokyo.ac.jp/en/oc/#utokyo_account のMFAページへの誘導から借りてきました．

### Courses

> Distributing teaching materials and submitting assignments on ITC-LMS are done on courses created for each year and class. Please refer to the following page for instructions on how to use it.

Reviewer 2: ```suggestion
Distributing teaching materials and submitting assignments on ITC-LMS take place in courses created for each year and class. Please refer to the following page for instructions on how to use it.
```
